tests/unit/providers/rhel/test-fixtures/snapshots/rhel:5/cve-2017-3526.json

```diff
@@ -45,7 +53,11 @@
             "NoAdvisory": true
           },
           "Version": "None",
```

Comment:
Does this fixture intentionally have a fix available date but no fixed version?

Reply:
nope that is an oversight